src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -369,8 +377,8 @@ type DotnetModuleConfig = {
     exports?: string[];
 } & Partial<EmscriptenModule>;
 type APIType = {
-    runMain: (mainAssemblyName: string, args: string[]) => Promise<number>;
-    runMainAndExit: (mainAssemblyName: string, args: string[]) => Promise<number>;
+    runMain: (mainAssemblyName: string, args?: string[]) => Promise<number>;
```

Comment:
we should specify somewhere what happens if you put args in both the config and in the args parameter to runMain. (concat the lists in config + explicit order?)

Reply:
method will win

Reply:
will update the description